src/landing-page/components/LandingPage/index.tsx

```diff
@@ -0,0 +1,301 @@
+import { faUpRightFromSquare } from "@fortawesome/free-solid-svg-icons";
```

Comment:
Compare with https://github.com/allen-cell-animated/nucmorph-colorizer/blob/main/src/routes/LandingPage.tsx if desired. This change is directly copied from it, with a few minor changes:
- Removed Link components on the "Load" buttons since it's currently unclear how load actions will work.
- Updates the feature highlight copy and title description based on designs.
- Commented out header component; currently unimplemented.